device/isis/segmentrouting.yaml

```diff
@@ -0,0 +1,289 @@
+components:
+  schemas:
+    Isis.SegmentRouting:
```

Comment:
what would be the default behavior of this `Isis.SegmentRouting` property under `isis` object. Theoretically whenever we will initialize  `isis` then   `Isis.SegmentRouting` property property also configure. So, we will end with enable `SegmentRouting` by default.

Reply:
Please ignore this command, we will handle it using Has